marklogic-data-hub-central/src/test/java/com/marklogic/hub/central/controllers/FlowControllerTest.java

```diff
@@ -126,6 +126,7 @@ void test() throws Exception {
                 .andDo(result -> {
                     JsonNode response = parseJsonResponse(result);
                 assertEquals(mappingInfo.targetEntityType, response.path("stepResponses").path("1").path("targetEntityType").asText(), "Info for the step should specify the Target Entity Type; response: " + response);
+                assertEquals("final", response.path("stepResponses").path("1").path("targetDatabase").asText(), "Info for the step should specify the target database; response: " + response);
```

Comment:
Is this always true? Or is it only true if targetDatabase is configured in the step options?

Reply:
It's only true if targetDatabase is configured in step options.

Reply:
Can you add that to the assertion message? The current assertion message is restating what the assertEquals is communicating - that we expect the stepResponse to have the targetDatabase. The assertion message needs to explain why that is - in this case, the reason is that if the step options contain targetDatabase, then it's expected to be added to the stepResponse.